Cargo.toml

```diff
@@ -22,7 +22,11 @@ itertools = "*"
 log = "*"
 env_logger = "*"
 sodiumoxide = "*"
-crust = "0.2.*"
+#crust = "0.2.*"
+
+[dependencies.crust]
+branch = "pub-proto"
```

Comment:
Changed to point to maidsafe's git repo, going to discuss with Viv adding it to crates.